tox.ini

```diff
@@ -81,7 +93,7 @@ commands =
     autopep8 --in-place --recursive --aggressive drf_yasg_json_api tests
 
 [testenv:dev]
-basepython = python3
+basepython = python3.8
```

Comment:
Please keep `python3`

Reply:
ok, btw i changed it initially because of warnings like this when running tox locally

```
UserWarning: conflicting basepython version (set 3.9, should be 3.8) for env 'dev-py38-django111-drf38-jsonapi28-yasg16';resolve conflict or set ignore_basepython_conflict
```

Maybe `ignore_basepython_conflict = true` can be added to `tox.ini` ?

Reply:
ok, thanks

I don't know which command you have run but, actually you could just choose the right python version by running:
```
tox -e dev-py38-...
```
If it solves the issue for your case I'll add some development notes to readme

Reply:
I was using `tox --parallel auto` to run all environments in parallel. 

Even if specific env is selected I still get the warnings.

```
tox -e dev-py38-django111-drf38-jsonapi28-yasg16  
drf-yasg-json-api/venv/lib/python3.9/site-packages/tox/config/__init__.py:592: UserWarning: conflicting basepython version (set 3.9, should be 3.8) for env 'dev-py38-django111-drf38-jsonapi28-yasg16';resolve conflict or set ignore_basepython_conflict
  warnings.warn(
drf-yasg-json-api/venv/lib/python3.9/site-packages/tox/config/__init__.py:592: UserWarning: conflicting basepython version (set 3.9, should be 3.8) for env 'dev-py38-django22-drf311-jsonapi30-yasg171';resolve conflict or set ignore_basepython_conflict
  warnings.warn(
drf-yasg-json-api/venv/lib/python3.9/site-packages/tox/config/__init__.py:592: UserWarning: conflicting basepython version (set 3.9, should be 3.8) for env 'dev-py38-django31-drf312-jsonapi40-yasg200';resolve conflict or set ignore_basepython_conflict
  warnings.warn(
drf-yasg-json-api/venv/lib/python3.9/site-packages/tox/config/__init__.py:592: UserWarning: conflicting basepython version (set 3.9, should be 3.8) for env 'dev-py38-django30-drf311-jsonapi32-yasg171';resolve conflict or set ignore_basepython_conflict
  warnings.warn(
```

Since this is just a warning, I don't think that this is so important, you can ignore it

Reply:
It looks like you have python 3.9 installed, so running `tox -e dev-py39-...`, but anyway something is to be fixed there... someday ;)